spec/acceptance/healthcheck_spec.rb

```diff
@@ -86,38 +86,44 @@
 
         context 'C99808 - --render-as yaml' do
           before(:all) do
+            on(master, puppet("cert list --all")) do |result|
+              @certs = result.stdout.scan(/\) ([A-F0-9:]+) /)
+            end
             @result = on(master, "puppet certregen healthcheck --all --render-as yaml")
+            @yaml = YAML.load(@result.stdout)
           end
           it 'should return valid yaml' do
             expect(YAML.parse(@result.stdout)).to be_instance_of(Psych::Nodes::Document)
           end
           it 'should contain expiration data for ca cert' do
-            pending("MODULES-4542")
-            expect(YAML.load(@result.stdout)).to match(/"ca".*\n\s+Status:.*\n\s+Expir/)
+            ca = @yaml.find { |record| record[:name] == 'ca' }
+            expect(ca).not_to be nil
+            expect(ca[:expiry][:status]).to eq(:expired)
           end
           it 'should contain expiration data for all node certs' do
-            pending("MODULES-4542")
             @certs.each do |cert|
-              expect(YAML.load(@result.stdout)).to include cert[0]
+              expect(@yaml.find { |record| record[:digest] =~ /#{cert[0]}/ }).not_to be nil
```

Comment:
Style nit: `cert` no longer references a cert, it looks like it's referencing a pair of cert and digest information. This variable name is misleading. I'll merge it as-is, but misleading variable names can be a surprisingly costly form of tech debt.